IdentityCore/src/requests/broker/MSIDSSOTokenResponseHandler.m

```diff
@@ -39,8 +39,6 @@ - (void)handleOperationResponse:(MSIDBrokerOperationTokenResponse *)operationRes
                           error:(NSError *)error
                 completionBlock:(MSIDRequestCompletionBlock)completionBlock
 {
-    if (!operationResponse) return;
```

Comment:
why this was removed?

Reply:
It prevented handling a case when we have error and there is no operationResponse.

Reply:
Thanks for clarification